content/10.blackbox.md

```diff
@@ -1,17 +1,15 @@
 ## Tip 8: Your DL models can be more transparent {#blackbox}
 
-In ML, interpretability refers to the study of the discriminative features used for classification or regression task.
-ML models can vary in terms of interpretability from a “transparent” to a “black-box” model, the first with a clear description of features importance found, for example, in common random forests implementations [@doi:10.1023/A:1010933404324]. 
-The second for the most widely used DL implementations.  
-Because of the large number of parameters and non-linear relationships among features, DL models are hard to interpret when compared to other ML models. 
+In ML, a model's interpretability refers to the ability to identify the discriminative features used for learning and that influence or sway the predictions.
```

Comment:
```suggestion
In ML, a model's interpretability refers to the ability to identify the discriminative features that influence or sway the predictions.
```
Does it make sense to further constrain this to _within the space of the predictions_? For models with nonlinearities - in some ranges of input a feature might greatly influence results, but that may be irrelevant with the space of inputs used for predictions.

Reply:
>that may be irrelevant with the space of inputs used for predictions.
Yes, I think your suggestion is fair, although I do think it can be interesting to see how the model fares on inputs far from the expected input space.